torch/_meta_registrations.py

```diff
@@ -4781,7 +4783,9 @@ def meta_select(self, dim, index):
     size = self.size(dim)
 
     torch._check_index(
-        not (-index > size or index >= size),
```

Comment:
same with this?

Reply:
yes, it is needed for NJT symbolic shapes processing for the test_case. There is njt index_put which does split underneath.